core/bones/password.py

```diff
@@ -69,21 +63,58 @@ class PasswordBone(StringBone):
         "core.bones.password.tooWeakMessage",
         defaultText="The entered password is too weak."
     )
+    _password_tests: List[Tuple] = [
+        (r"(?=.*[A-Z])", translate("core.bones.password.no_capital_letters",
+                                   defaultText="The password entered has no capital letters.")),
+        (r"(?=.*[a-z])", translate("core.bones.password.no_lowercase_letters",
+                                   defaultText="The password entered has no lowercase letters.")),
+        (r"(?=.*\d)", translate("core.bones.password.no_digits",
+                                defaultText="The password entered has no digits.")),
+        (r"(?=.*\W)", translate("core.bones.password.no_special_characters",
+                                defaultText="The password entered has no special characters.")),
+
+    ]
+
+    def __init__(
+        self,
+        *,
+        min_password_length: int = 8,
+        test_threshold: int = 3,
+        password_tests: List[Tuple] = None,
+        **kwargs
+    ):
+        """
+            Initializes a new Password Bone.
+
+            :param min_password_length: The minimum length of the password all passwords with a length that is
+                    smaller this will be invalid.
```

Comment:
I'm not a English pro, but i think this sound better.

```suggestion
            :param min_password_length: The minimum length of a password.
                Passwords with a shorter length will be invalid.
```